tv_apps/smart-system/js/permission_manager.js

```diff
@@ -98,6 +106,16 @@
         evt.preventDefault();
       });
 
+      // Initialize key spatial navigator
+      this._spatialNavigator = new SpatialNavigator([
+        this.yes, this.no, this.moreInfoLink, this.hideInfoLink,
+        this.rememberSection]);
+      this._spatialNavigator.straightOnly = true;
+      this._spatialNavigator.ignoreHiddenElement = true;
```

Comment:
I recommend to set `rememberSource = true` as well. It would be better to round trip among buttons and "more info" item.